code/src/App.js

```diff
@@ -1,9 +1,78 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import ThoughtForm from './components/ThoughtsForm'
+import ThoughtList from './components/ThoughtList'
+
+import { API_URL, LIKES_URL } from './reusable/urls'
+
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([])
+  const [thoughtNew, setThoughtNew] = useState('')
+
+  useEffect(() => {
+    fetchThoughtList()
```

Comment:
I like that you made the fetch a function, makes for easier reading and keeping track of whats happening.